src/reflect/all_test.go

```diff
@@ -127,7 +127,7 @@ var deepEqualTests = []DeepEqualTest{
 	{&loopy1, &loopy1, true},
 	{&loopy1, &loopy2, true},
 	//{&cycleMap1, &cycleMap2, true},
-	{&cycleMap1, &cycleMap3, false},
+	//{&cycleMap1, &cycleMap3, false},
```

Comment:
Yes, this bug was causing the maps equality code to thing the maps were different before it actually got to comparing the map keys, when it would the fail with unimplemented `reflect.Value.MapKeys()` call.